vitaliy_dobrovichan/task2/movie_parser.rb

```diff
@@ -0,0 +1,35 @@
+class MovieParser
+  attr_reader   :address, :movie_names
+  attr_accessor :aggregated_data
+
+  def initialize(address, movie_names)
+    @address     = address
+    @movie_names = movie_names
+    @browser     = Watir::Browser.new
+  end
+
+  def search_movie(query)
+    search_input = @browser.text_field(name: "kp_query")
+    search_input.set query
+  end
+
+  def parse
+    aggregated_data = []
+    @browser.goto address
+
+    sleep 2
```

Comment:
good approach is to use newline delimiter to split **logical** blocks, sleep doesn't need it, imho ;)